bundle/pipeline.go

```diff
@@ -61,7 +61,7 @@ func (b *Bundle) CreatePipeline(req interface{}) (*apistructs.PipelineDTO, error
 		return nil, apierrors.ErrInvoke.InternalError(err)
 	}
 	if !resp.IsOK() || !createResp.Success {
-		return nil, toAPIError(resp.StatusCode(), createResp.Error)
+		return nil, toAPIError(resp.StatusCode(), createResp.Error).SetCtx(createResp.Error.Ctx)
```

Comment:
the args[1] of `toAPIError` is `createResp.Error`, which already contains Ctx.

Reply:
![image](https://user-images.githubusercontent.com/30427474/177294947-73803a90-0f92-4c54-9927-0f1c221d19dd.png)
adjusted `toAPIError`, add WithCtx  in `New` func